client/app/states/login/login.state.js

```diff
@@ -36,6 +36,10 @@
       password: API_PASSWORD,
     };
 
+    if ($window.location.href.includes("?timeout")) {
+      Notifications.message('danger', '', __('Your session has timed out.'), true);
```

Comment:
Oh snap, really?  Yeah yeah, this makes sense.  Ignore my previous comment then, all good with this, great job! 🌮 

@chriskacerguis  👍